### PR TITLE
🔥 Add Joystick control

### PR DESCRIPTION
**Pull request by ssakone**

### CODE REVIEW

File: qmlcomps/videocamera/Light.qml [hunk 1]
```diff
@@ -11,13 +11,13 @@ Item {
...
         model: 36
-        SparseLight {
+        Rectangle {
             property double angle: 2 * index * Math.PI / 36
             radius: 12 / 660 * root.radius
-            lightOn: false
-            lightBrightness: 0
-            x: root.radius - radius + (root.radius - radius) * Math.cos(angle)
-            y: root.radius - radius + (root.radius - radius) * Math.sin(angle)
+//            lightOn: false
```
Comment (nrbnlulu) [inline qmlcomps/videocamera/Light.qml]: Why is that commented?

File: qmlcomps/videocamera/Joystick.qml [hunk 2]
```diff
@@ -80,23 +83,30 @@ Item {
     MouseArea {
```
Comment (nrbnlulu) [inline qmlcomps/videocamera/Joystick.qml:86]: This can use a `DragHandler` and `WheelHandler`?

File: qmlcomps/videocamera/joystickersignaler.h [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#ifndef JOYSTICKERSIGNALER_H
```
Comment (nrbnlulu) [inline qmlcomps/videocamera/joystickersignaler.h:1]: `#pragma once` is not cleaner?

File: qmlcomps/videocamera/Handle.qml [hunk 4]
```diff
@@ -1,11 +1,11 @@
-import QtQuick
+import QtQuick 2.0
```
Comment (nrbnlulu) [inline qmlcomps/videocamera/Handle.qml]: why 2.0?

File: qmlcomps/videocamera/Groove.qml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import QtQuick 2.0
```
Comment (nrbnlulu) [inline qmlcomps/videocamera/Groove.qml]: 2.0?

File: qmlcomps/videocamera/Light.qml (hunk 1 above)
Comment (ssakone) [reply qmlcomps/videocamera/Light.qml]: no need

File: qmlcomps/videocamera/Joystick.qml (hunk 2 above)
Comment (ssakone) [reply qmlcomps/videocamera/Joystick.qml:86]: Mouse handler was already implemented so i just get wheel event

File: qmlcomps/videocamera/joystickersignaler.h (hunk 3 above)
Comment (ssakone) [reply qmlcomps/videocamera/joystickersignaler.h:1]: i don't understand

File: qmlcomps/videocamera/Handle.qml (hunk 4 above)
Comment (ssakone) [reply qmlcomps/videocamera/Handle.qml]: it was already in the file i take directly from original repo, that as no effect in the app

File: qmlcomps/videocamera/Groove.qml (hunk 5 above)
Comment (ssakone) [reply qmlcomps/videocamera/Groove.qml]: it was already in the file i take directly from original repo, that as no effect in the app

File: qmlcomps/videocamera/Light.qml (hunk 1 above)
Comment (ssakone) [reply qmlcomps/videocamera/Light.qml]: this file is not used

**Pull request merged**